addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -6,48 +6,23 @@
     .o_notebook .tab-content .tab-pane .o_mail_body {
         // cancel the padding of the form_sheet
         margin-top: -$o-sheet-cancel-hpadding;
-        margin-left: -$o-sheet-cancel-hpadding;
-        margin-right: -$o-sheet-cancel-hpadding;
+        margin-left: calc(var(--formView-sheet-padding-x) * -1);
```

Comment:
I think you could use the `o-form-sheet-negative-margin` mixin instead of raw css.